package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "keypair_lib",
+  "version": "1.0.0",
+  "description": "Keypair library",
+  "main": "lib/index.js",
+  "scripts": {
+    "test": "jest --config jestconfig.json",
+    "format": "prettier --write \"src/**/*.ts\"",
+    "lint": "tslint -p tsconfig.json",
+    "prepare": "npm run build",
+    "prepublishOnly": "npm test && npm run lint",
+    "preversion": "npm run lint",
+    "version": "npm run format && git add -A src",
+    "postversion": "git push && git push --tags",
+    "build": "npm run clean && tsc && npm run copy-files",
+    "clean": "rm -rf ./lib",
+    "copy-files": "cp -r ./zencode/ ./lib/zencode && cp -r ./props/ ./lib/props"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/LedgerProject/keypairoom.git"
+  },
+  "keywords": [
+    "keypair",
+    "zencode"
+  ],
+  "author": "",
+  "license": "ISC",
```

Comment:
I would prefer to avoid using ISC as per https://en.wikipedia.org/wiki/ISC_license#Reception and in case, the terms are missing.

Reply:
Idk what to put there (it's just default license you get with npm init) even license. Is there anything from your repo I can copy ?

Reply:
MIT should be fine then ;)

Reply:
changed it to MIT